otherlibs/configurator/dune

```diff
@@ -1,3 +1,3 @@
 (env
  (_
-  (flags :standard \ -alert=-unstable)))
+  (flags :standard \ -alert -unstable)))
```

Comment:
That'll work for this case, but in general it means: exclude "-alert" and exclude "-unstable" which is not the same as exclude "-alert -unstable". It'll work for us as "-unstable" doesn't appear anywhere else.

Reply:
Yes, removing flags in this way is unsafe but I guess we don't have a safer mechanism. Hopefully, it'll not bite us in this particular case.

Reply:
If the intent is to **enable** the `unstable` alert, I think you can use `:standard -alert +unstable`.

Reply:
The intent is to remove the flag because configurator needs to build with 4.02 and 4.02 doesn't know the `-alert` flag.

Reply:
Ah right, then it doesn't help, sorry for the noise.

Reply:
OK, merging as is then.